python_modules/dagster/dagster/components/core/context.py

```diff
@@ -22,6 +22,7 @@ class ComponentLoadContext:
     """Context for loading a single component."""
 
     path: Path
+    project_path: Path
```

Comment:
should it be `project_root` ?

Reply:
Do you think it should named `pojrect_root` throughout, including in the templating?

Reply:
i think so, not with great conviction. Line of thought is just that `project_root` is clearer than `project_path` about what it means, the latter possibly interpreted as path within project or something

Reply:
👍🏻